src/ast/ast_enrich.c

```diff
@@ -261,6 +261,154 @@ void AST_AttachName(AST *ast, const cypher_astnode_t *node, const char *name) {
 	cypher_astnode_attach_annotation(ast->name_ctx, node, (void *)name, NULL);
 }
 
+//------------------------------------------------------------------------------
+//  Annotation context - Attach named path projection identifiers their respective ast structure
+//------------------------------------------------------------------------------
+
+static void _FreeProjectNamedPathAnnotationCallback(void *unused, const cypher_astnode_t *node,
+													void *annotation) {
+	// Do nothing, annotation is a pointer to another part of the ast.
+}
+
+static AnnotationCtx *_AST_NewProjectNamedPathContext(void) {
+	AnnotationCtx *project_all_ctx = cypher_ast_annotation_context();
+	cypher_ast_annotation_context_release_handler_t handler = &_FreeProjectNamedPathAnnotationCallback;
+	cypher_ast_annotation_context_set_release_handler(project_all_ctx, handler, NULL);
```

Comment:
The handler is optional, so these lines and `_FreeProjectNamedPathAnnotationCallback` can be deleted.

Reply:
thanks
done